Area51/Sources/SubredditsViewController.swift

```diff
@@ -16,28 +25,63 @@ class SubredditsViewController: UIViewController {
     private func setupTableView() {
         self.tableView.delegate = self
         self.tableView.dataSource = self
+        self.tableView.refreshControl = UIRefreshControl()
```

Comment:
I do not think we need the ability to refresh. The list should be pretty static across sessions. Can remove all refresh functionality.